docs/cloud/features/single_sign_on.md

```diff
@@ -0,0 +1,198 @@
+# SSO (Single sign-on) 
+
+## Overview
+
+Tobiko Cloud supports single sign-on (SSO) through OpenID and SAML 2.0 providers. 
+
+This makes it easy to provision access to users and simplifies authentication. 
+
+
+## Setup & Prerequsites 
+
+You must have an active Tobiko Cloud instance with SSO enabled. Please contact your account team to ensure this is enabled. 
+
+If your Tobiko Cloud instance is setup to require SSO, then you won't need to provide a token in your `tcloud.yml` configuration. 
+
+Below is an example of a `tcloud.yml` configuration:
+```yaml
+projects:
+    <Project name>:
+        url: <The project URL>
+        token: <The access token>
+        gateway: <The name of the SQLMesh gateway to use with this project>
+        extras: <Optional - Any extras that should be installed with sqlmesh-enterprise>
+        pip_executable: <Optional - The path to the pip executable to use. Ex: `uv pip` or `pip3`. Must install packages to the python environment running the tcloud command>
+default_project: <The name of a project to use by default>
+```
+
+## Identity Providers
+
+Tobiko Cloud currently supports OpenID and SAML 2.0. 
+
+### OpenID
+
+This provider implements [OpenID Connect Core
+1.0](https://openid.net/specs/openid-connect-core-1_0.html) in order to allow us
+to login with most OAuth2 login providers.
+
+There are two types of customers that might use OpenID Providers. The first is a
+customer that would like to use a shared provider like Google, Github,
+Microsoft, etc. 
+
+#### Google OAuth
+
+To enable Google OAuth, all we need is your domain. From here, we can switch SSO on with Google OAuth. 
+
+The login flow will look like the following if you access cloud.tobikodata.com/auth/login directly from your browser. If authenticating through CLI see [here](https://sqlmesh.readthedocs.io/en/stable/cloud/features/single_sign_on/#status) for more details. 
+
+<video width="100%" height="100%" controls>
+  <source src="google_oauth.mp4" type="video/mp4">
```

Comment:
Make this a loom video embedded. 1.1mb for a video each time someone clones this repo isn't worth the performance penalty.